test/e2e/e2e.go

```diff
@@ -91,5 +91,11 @@ func RunE2ETests(t *testing.T) bool {
 		config.DefaultReporterConfig.SlowSpecThreshold = 45.0
 	}
 
+	// Set junit report path and filename if required by user
+	junitReport := framework.TestContext.JunitReport
+	if junitReport != "" {
+		config.DefaultReporterConfig.ReportFile = junitReport
+	}
```

Comment:
We could skip the additional variable; but we can even skip the conditional entirely, since if `JUnitReport` is empty, we just set `ReportFile` to empty:

```suggestion
	config.DefaultReporterConfig.ReportFile = framework.TestContext.JunitReport
```

Reply:
That's a good point.
Thanks.